filebeat/docs/getting-started.asciidoc

```diff
@@ -218,15 +218,23 @@ include::../../libbeat/docs/shared-logstash-config.asciidoc[]
 :allplatforms:
 include::../../libbeat/docs/shared-template-load.asciidoc[]
 
+[[load-kibana-dashboards]]
+=== Step 5: Set up the Kibana dashboards
+
+:allplatforms:
+include::../../libbeat/docs/dashboards.asciidoc[]
+
 [[filebeat-starting]]
-=== Step 5: Start Filebeat
+=== Step 6: Start Filebeat
 
 Start Filebeat by issuing the appropriate command for your platform.
 
 NOTE: If you use an init.d script to start Filebeat on deb or rpm, you can't
 specify command line flags (see <<command-line-options>>). To specify flags,
 start Filebeat in the foreground.
 
+//REVIEWERS: With the new command syntax, should the deb and rpm instructions here say run instead of start?
```

Comment:
No, for deb/rpm these commands remain the same. However for different reason we should change the commands to `sudo service filebeat start`. This ensures that service is started using a clean environment.